mount/mount_linux.go

```diff
@@ -107,6 +115,17 @@ func (m *Mount) Mount(target string) (err error) {
 		// Remount the bind to apply read only.
 		return unix.Mount("", target, "", uintptr(oflags|unix.MS_REMOUNT), "")
 	}
+
+	// The only remapping of both GID and UID is supported
+	if opt.mapuid != "" && opt.mapgid != "" {
+		// Remap GID UID of container rootfs from host to container
+		// to have proper ownership of rootfs files in container
+		// user namespace
+		if err := MapMount(opt.mapuid, opt.mapgid, target); err != nil {
+			return err
+		}
+	}
```

Comment:
Could this also allow remapping just one of them (`uid` or `gid`)? I could see that being useful for container platforms like k8s where the uid and gid can be separate parameters for the pod.

Reply:
@dobsonj In which case exactly only remapping one is useful?

Reply:
@dobsonj you can use the identity mapping if you don't want to remap one of them.